### DISCUSSION TITLE
`/var/opt/manifold/api/tmp/ingestion` folder gets large

### DISCUSSION THREAD

**Discussion by yl5682** [ManifoldScholar/manifold#3201] [Q&A] [answered]
I wonder is it safe to remove content in the `/var/opt/manifold/api/tmp/ingestion` folder? My folder gets large over time but I don't know if removing files in this folder will corrupt Manifold. Thank you!

**Comment by zdavis** [comment, marked as answer]
Yes, this should be safe. I'd expect Manifold to clean this up, so having files in there is a bug. I expect that ingestion is failing for some reason and triggering an uncaught exception, in which case Manifold might not remove the file. We could look into the failed ingestion, but we'd need more information and steps to reproduce.
